src/admin/__tests__/createAcls.spec.js

```diff
@@ -0,0 +1,228 @@
+const createAdmin = require('../index')
+// const { KafkaJSProtocolError } = require('../../errors')
+// const { createErrorFromCode } = require('../../protocol/error')
+
+const {
+  secureRandom,
+  createCluster,
+  newLogger,
+  sslConnectionOpts,
+  saslBrokers,
+} = require('testHelpers')
+
+const RESOURCE_TYPES = require('../../protocol/resourceTypes')
+const OPERATION_TYPES = require('../../protocol/operationsTypes')
+const PERMISSION_TYPES = require('../../protocol/permissionTypes')
+const RESOURCE_PATTERN_TYPES = require('../../protocol/resourcePatternTypes')
+
+describe('Admin', () => {
+  const topicName = `test-topic-${secureRandom()}`
+  let admin
+
+  afterEach(async () => {
+    await admin.disconnect()
+  })
+
+  describe('createAcls', () => {
+    test('create new topic and apply acl', async () => {
+      const saslConnectionOpts = () =>
```

Comment:
No need to assign a new connection options object here. Just pass in `saslConnectionOpts()` from testHelpers into `createCluster` directly.  Same as in https://github.com/tulios/kafkajs/blob/master/src/admin/__tests__/connection.spec.js#L28-L32 for example.